src/hash/rescue/rpo/digest.rs

```diff
@@ -66,6 +70,18 @@ impl Digest for RpoDigest {
 
         result
     }
+
+    fn from_random_bytes(buffer: &[u8]) -> Self {
```

Comment:
Do we still need this function? And if so, I would add a doc comment to it explaining that the resulting `Self` has a small bias (or something like that).

Reply:
Removed from here and, more importantly, from Winterfell and updated the code there